File: 30 Days of Code/Day 02/Operators.rb
meal_cost = gets.to_f

tip_percent = gets.chomp.to_f

tax_percent = gets.to_f

tip = meal_cost * tip_percent / 100
tax = meal_cost * tax_percent / 100
price = meal_cost + tip + tax
puts "#{price.round}\n"

File: 30 Days of Code/Day 02/Operators.cpp
void solve(double meal_cost, int tip_percent, int tax_percent) {
        // Get the Tax
        double tax = (meal_cost * tax_percent) / 100;
        // Get the Tip
        double tip = (meal_cost * tip_percent/100);
        // Total Cost
        double totalCost = meal_cost + tax + tip;
        // Round the total cost and output to console
        cout<<round(totalCost)<<endl;
}
